modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/BaseNotificationTypeTest.java

```diff
@@ -149,6 +149,42 @@ public static void setUpClass() throws Exception {
 			"textObjectField", RandomTestUtil.randomString()
 		).build();
 
+		siteScopeObjectEntryValues = LinkedHashMapBuilder.<String, Object>put(
+			"booleanObjectField", RandomTestUtil.randomBoolean()
+		).put(
+			"dateObjectField",
+			() -> {
+				SimpleDateFormat simpleDateFormat = new SimpleDateFormat(
+					"yyyy-MM-dd");
+
+				return simpleDateFormat.format(RandomTestUtil.nextDate());
+			}
+		).put(
+			"dateTimeObjectField",
+			() -> {
+				SimpleDateFormat simpleDateFormat = new SimpleDateFormat(
+					"yyyy-MM-dd 00:00:00.0");
+
+				return simpleDateFormat.format(RandomTestUtil.nextDate());
+			}
+		).put(
+			"emailTextObjectField",
+			StringUtil.toLowerCase(RandomTestUtil.randomString()) +
+				"@liferay.com"
+		).put(
+			"integerObjectField", RandomTestUtil.nextInt()
+		).put(
+			"picklistObjectField",
+			new ListEntry() {
+				{
+					key = listTypeEntry.getKey();
+					name = listTypeEntry.getName(LocaleUtil.US);
+				}
+			}
+		).put(
+			"textObjectField", RandomTestUtil.randomString()
+		).build();
```

Comment:
@rijema let's keep only the `textObjectField` because it's simple and we don't need to cover other terms that are already  covered.

Thank you!